Mage.Sets/src/mage/cards/c/Cowardice.java

```diff
@@ -68,7 +67,7 @@ public Cowardice copy() {
 class CowardiceTriggeredAbility extends TriggeredAbilityImpl {
 
     public CowardiceTriggeredAbility() {
-        super(Zone.BATTLEFIELD, new ReturnToHandTargetEffect());
+        super(Zone.BATTLEFIELD, new ReturnToHandTargetEffect(), false);
```

Comment:
This doesn't need to be pass three arguments unless you are declaring optional to be true.